NEWS

```diff
@@ -1,3 +1,36 @@
-Changes in version 0.99.0 (2021-09-29)
+# Changes (2022-05-30)
```

Comment:
This should contain the version number at which the changes apply. See model from above (deleted) line